act-rules-format.bs

```diff
@@ -451,63 +494,94 @@ Definitions {#definitions}
   <dd>
     <p>A requirement is a condition that has to be satisfied in order to conform to a standard, or to comply with a contract, policy or regulation. An accessibility requirement is a requirement aimed at improving access for people with disabilities to an ICT product.</p>
-    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that include recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
+    <p>A common example of accessibility requirements are the WCAG 2.1 success criteria. There are other standards, including W3C standards that have recommendations for accessibility, such as WAI-ARIA and HTML. Accessibility requirements are also often found in company policies, regional standards or in legislation.</p>
   </dd>
 
   <dt><dfn>Accessibility requirements document</dfn><dt>
   <dd>
-    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requiremenst=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+    <p>A document, such as a standard, contract, policy or regulation, that includes [=accessibility requirements=]. For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, BBC HTML Accessibility Standards v2.0</p>
+  </dd>
+
+  <dt><dfn>Finding</dfn></dt>
+  <dd>
+    <p>When evaluating a [=test subject=] using a rule, the findings of that rule are:</p>
+    <ul>
+      <li>All [=test targets=] in the test subject</li>
+      <li>Which test targets met which [expectations](#expectations) in the test subject</li>
+      <li>The [=outcome=] (passed, failed or inapplicble) for the test subject</li>
+    </ul>
+  </dd>
+
+  <dt><dfn>meet expectations</dfn></dt>
+  <dd>
+    <p>A [=test target=] meets an expectation, when that expectation is true for the test target individually, or when the test target increases a count that is required for the expectation to be true.</p>
+    <div class=example>
+      <p>**Example:** An HTML `img` element an `alt` attribute meet the following expectation:</p>
+      <blockquote>Each test target has an alt attribute</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML element with a duplicate ID does not meet the following expectation:</p>
+      <blockquote>No test target has the same ID</blockquote>
+    </div>
+    <div class=example>
+      <p>**Example:** An HTML `video` element without captions does not meet the following expectation:</p>
+      <blockquote>80% of all test targets has a caption</blockquote>
+    </div>
+    <div class=note>
+      <p>Expectations are true or false assertions about a test subject. Individual test targets can meet all expectations, even if some expectations are false about the test subject as a whole</p>
+    </div>
+  </dd>
+
+  <dt><dfn>Outcome</dfn></dt>
```

Comment:
**Note:** A rule has one passed of failed outcome for every test target. When there are no test targets the rule has one inappliable outcome. This means that each test subject will have one or more outcomes.